imports/api/jccc-finances.js

```diff
@@ -0,0 +1,60 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+
+
+const JCCCFinances = new Mongo.Collection('jccc-finances');
+JCCCFinances.schema = new SimpleSchema({
+    "applicationID": {
+        type: String,
+        label: "JCCC Application ID"
+    },
+    "totalTransaction": {
+        type: Number,
+        decimal: true,
+        label: "Total Transaction Amount"
+    },
+    "ccTransaction": {
+        type: Number,
+        decimal: true,
+        label: "CC Transaction Amount"
+    },
+    "seasTransaction": {
+        type: Number,
+        decimal: true,
+        label: "SEAS Transaction Amount"
+    },
+    "gsTransaction": {
+        type: Number,
+        decimal: true,
+        label: "GS Transaction Amount"
+    },
+    "bcTransaction": {
+        type: Number,
+        decimal: true,
+        label: "BC Transaction Amount"
+    },
+    "receiptAmount": {
```

Comment:
Is this the sum of CC, SEAS, GS, and BC transactions? If so, is there a way to set up validate any number put in here to correspond with the other amounts or to calculate this automatically based on the other entries?